jenkins/build.sh

```diff
@@ -15,32 +15,43 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
-
 set -ex
 BASE_DIR=$(realpath "$(dirname $0)/..")
 PROG_NAME=$(basename $0)
 DIST=$(realpath $BASE_DIR/dist)
 RPM_NAME="cortx-ha"
 CORTX="cortx"
 HA_PATH="/opt/seagate/${CORTX}"
+HA1="1"
+HA2="2"
+MINOR_VERSION="0"
+REVISION_VERSION="0"
 
 usage() {
     echo """
-usage: $PROG_NAME [-v <coretx-ha version>] [-d]
-                            [-b <build no>] [-k <key>]
+usage: $PROG_NAME [-v <cortx-ha major version>] [-m <cortx-ha minor version>]
+                  [-r <cortx-ha revision version>] [-b <build no>] [-k <key>]
 
 Options:
-    -v : Build rpm with version
+    -v : Build rpm with major version
+    -m : Build rpm with minor version
+    -r : Build rpm with revision version
     -b : Build rpm with build number
     -k : Provide key for encryption of code
```

Comment:
Please modify the description of -k as discussed in the call.
Add "used by pyinstaller".

Reply:
done